docker-compose.yaml

```diff
@@ -1,18 +1,21 @@
+version: '3'
 services:
   reverse-proxy:
-    image: traefik:2.5
+    image: traefik:latest
     command: --api.insecure=true --providers.docker=true --entrypoints.web.address=:80
     ports:
       - "80:80"
       - "8080:8080"
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
   mongodb_seed:
-    image: bitnami/mongodb:4.2
+    image: bitnami/mongodb:latest
```

Comment:
same as above + I think now the official mongo image is simply "mongo", which might require small changes in the mongo_import.sh. As a general comment, I would try to have 2 PRs, one for migrating to the new be, and another to upgrade the containers and modify the docker-compose.

Reply:
Yes, better pin at least the major version: `bitnami/mongodb:4`

But we should consider upgrading to a recent version here. Our instances are running with bitnami/mongodb:14.9 just fine.

Reply:
I agree! I also noticed that the officially supported one is here now: https://hub.docker.com/_/mongo